### ISSUE TITLE
Strange behavior when deleting file named ""

### ISSUE THREAD

**Issue by akrejczinger** [PhrozenByte/rmtrash#8]
When deleting a file named "" with rmtrash, the current working directory is moved to trash instead.
Steps to reproduce:
```
$ mkdir ~/tmp
$ cd ~/tmp
$ touch '""'
$ rmtrash '""'

# open new shell
$ pwd
<userhome>/.local/share/Trash/files/tmp_3
$ ls
""
```

**Comment by akrejczinger** [comment]
Version used:
rmtrash 1.11 (build 20150511)

**Comment by PhrozenByte** [comment]
Indeed. Fixed in [v1.12](https://github.com/PhrozenByte/rmtrash/releases/tag/v1.12) 🎉

Thank you for reporting this @akrejczinger! 👍
